exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/FileSelection.java

```diff
@@ -402,4 +402,18 @@ public String getCacheFileRoot() {
     return cacheFileRoot;
   }
 
+  @Override
+  public String toString() {
+    final StringBuilder sb = new StringBuilder();
+    sb.append("root=" + this.selectionRoot);
+
+    sb.append("files=[");
+    for (final String file : this.files) {
+      sb.append(file);
```

Comment:
It's used only internally for now, since Drill's Explain output only generates physical plan. 

You suggestion makes sense. I add a comma after each file name.